charts/dotcms/values.yaml

```
@@ -303,7 +303,7 @@ envVarsDefaults:
   # @default -- 0.0.0.0/0
   DOT_SYSTEM_STATUS_API_IP_ACL: "0.0.0.0/0"
   # -- List of IPs/subnets to block from making remote calls
-  DOT_REMOTE_CALL_SUBNET_BLACKLIST: "169.254.169.254/32127.0.0.1/3210.0.0.0/8172.16.0.0/12192.168.0.0/16"
+  DOT_REMOTE_CALL_SUBNET_BLACKLIST: "169.254.169.254/32,127.0.0.1/32,10.0.0.0/8,172.16.0.0/12,192.168.0.0/16"
```

Not needed right now because it's beyond scope of this pr.  However, we should be moving to more inclusive language in our codebase.  `DENYLIST` is more meaningful description and more inclusive 😄 
   
```suggestion
  DOT_REMOTE_CALL_SUBNET_DENYLIST: "169.254.169.254/32,127.0.0.1/32,10.0.0.0/8,172.16.0.0/12,192.168.0.0/16"
```

---

You're absolutely spot on about moving toward more inclusive language, this is definitely the right direction to go. 

However, there's a deeper technical consideration here. After diving into the dotCMS core codebase, I've discovered that this change has significant implications at the application level.

This isn't just a Helm chart change, it requires coordinated changes across:
- dotCMS core application code
- Default configuration properties
- Documentation and migration guides
- Backward compatibility handling (Most important).

Rather than making a change that could introduce security vulnerabilities, I suggest we:

*Short term (Current PR):*

- Keep DOT_REMOTE_CALL_SUBNET_BLACKLIST to maintain functionality
- Add comprehensive documentation about the planned transition

*Medium term (Coordinate with dotCMS team):*

- Propose this change to the dotCMS core team for proper implementation
- Implement dual-variable support in core (DENYLIST takes precedence, BLACKLIST as fallback)
- Add deprecation warnings for the old variable

*Long term (Future versions):*

- Phase out BLACKLIST once core support is established
- Complete transition to inclusive language